domains.txt

```diff
@@ -175,12 +199,17 @@ digitalgurus.co.uk
 digitalpursuit.co.uk
 digitechis.com
 digitechresourcing.co.uk
+digvis.co.uk
 direct-recruitment.co.uk
 dnsrecruitment.co.uk
 dpconnect.co.uk
+dsemploymentgroup.co.uk
 durlstonpartners.com
 e-recruiter.co.uk
 e-resourcing.be
+e-resourcing.co.uk
```

Comment:
E-Resourcing successfully delivers recruitment services backed by professionals with an average of over 10 years industry experience. The company combines traditional ‘best in class' recruitment practices with state-of-the-art web and database search skills.